src/Entity/ContentMagicTraits.php

```diff
@@ -68,8 +68,8 @@ public function magicEditLink()
     public function magicTitleFields(): array
```

Comment:
does this method always return an array with single value? why?

Reply:
Because in `contenttypes.yml`, we can expect users to put a variety of stuff, like 

```
title_format: title
title_format: [ title ]
title_format: [ first, last ]
title_format: [ id, slug, title ]
```

If we always return an array, we need less checks in the places we process it.